utreexo-validation-bip.md

```diff
@@ -54,36 +86,41 @@ Below outlines the data that was used to calculate those hashes:
 | scriptPubkey size | varint                   | scriptPubKey length in bytes              |
 | scriptPubkey      | variable byte array      | The locking script of the UTXO            |
 
+Each field being defined as follows:
+
 ##### Version tag
+
 We use tagged hashes for the hashes committed in the accumulator for versioning
 purposes. This is added so that if there are changes in the preimage of the
 hash, the version tag helps to avoid misinterpretation.
 
-The Utreexo version tag is the sha512 hash of the string "UtreexoV1". The string
-"UtreexoV1" as a byte slice is "[85 116 114 101 101 120 111 86 49]". As a hex
-string, it's "5574726565786f5631".  (The resulting 64 byte output is 
-5b832db8ca26c25be1c542d6cceddda8c145615cff5c35727fb3462610807e20ae534dc3f64299199931772e03787d18156eb3151e0ed1b3098bdc8445861885)
-(right?)
+The Utreexo version tag is the SHA512 hash of the string `UtreexoV1`, which is represented as the slice 
```

Comment:
Since we say vector in a different part of the bip, we should probably say vector here instead of slice.